ui/apps/platform/src/Containers/AccessControl/PermissionSets/PermissionSetsList.tsx

```diff
@@ -77,13 +79,14 @@ function PermissionSetsList({
                     <Thead>
                         <Tr>
                             <Th width={20}>Name</Th>
+                            <Th width={15}>Origin</Th>
```

Comment:
The percents add up to more than 100. Do we need to deduct from one of the other columns?